modules/builders-virtual-environment.adoc

```diff
@@ -81,7 +81,7 @@ This action requires cluster admin privileges. This is required because builders
 +
 [source,terminal]
 ----
-$ oc sa get-token -n virtual-builders quay-builder
+$ oc create token quay-builder -n virtual-builders
```

Comment:
Do we need to support OCP 4.10- and OCP 4.11 at the same time.
If so, is it better to describe like below
```
* OCP 4.10 and earlier
oc sa get-token -n virtual-builders quay-builder
* OCP 4.11
oc create token quay-builder -n virtual-builders
```

Reply:
agree to give quay customers clear instructions whether they're running on OCP 4.10 or 4.11.